test/vtcomposite.test.js

```diff
@@ -6,6 +6,16 @@ var zlib = require('zlib');
 var mvtFixtures = require('@mapbox/mvt-fixtures');
 var vt = require('@mapbox/vector-tile').VectorTile;
 var pbf = require('pbf');
+var geoData = require('./fixtures/four-points.js');
+
+
+function long2tile(lon,zoom) { 
```

Comment:
@millzpaugh [sphericalmercator](https://github.com/mapbox/sphericalmercator#pxll-zoom) might have functions that'll do this for you!

Reply:
If these aren't available (which is entirely possible) it'd be an awesome addition to the library, which I think we should considering doing to simplify the tests in this library. 

Long-term benefit: What if we learn that `lat2tile` is incorrect? We should ensure all functions we are using are reliably tested!